### PR TITLE
Minor edits to service resource's scheduled_actions block

### PR DESCRIPTION
**Pull request by katkasian**
Adds minor edits to [service resource ](https://registry.terraform.io/providers/PagerDuty/pagerduty/latest/docs/resources/service) documentation, mainly focusing on `scheduled_actions` block.

### CODE REVIEW

**Comment by chai-srini** [comment]
@katkasian Are you able to provision a Service with `"use_support_hours"` in `incident_urgency_rule` and not specify any `scheduled_actions`?

**Comment by stmcallister** [comment]
Hi @katkasian! I checked with the PagerDuty engineering team and they verified that, "`scheduled_actions` is required when the `incident_urgency_rule` is `use_support_hours`, and forbidden otherwise."

File: website/docs/r/service.html.markdown [hunk 1]
```diff
@@ -87,7 +87,7 @@ The block contains the following arguments:
   * `start_time` - The support hours' starting time of day.
   * `end_time` - The support hours' ending time of day.
 
-When using `type = "use_support_hours"` in `incident_urgency_rule` you must specify at least one (otherwise optional) `scheduled_actions` block.
+When using `type = "use_support_hours"` in `incident_urgency_rule` you can specify an optional `scheduled_actions` block.
```
Comment (stmcallister) [inline website/docs/r/service.html.markdown:90]: Based on the information I received from PagerDuty Engineering, the new language should be something more like, 

When using `type = "use_support_hours"` in `incident_urgency_rule` a `scheduled_actions` block is required.

**Review by stmcallister — CHANGES_REQUESTED** [review]

File: website/docs/r/service.html.markdown (hunk 1 above)
Comment (katkasian) [reply website/docs/r/service.html.markdown]: Thank you @stmcallister ! Reflected your comment.

**Review by stmcallister — APPROVED** [review]
👍 Thank you!

**Pull request merged**